tasks/install-on-Darwin.yml

```diff
@@ -29,3 +36,12 @@
   when: buildkite_agent_load_bash_profile == True
   notify:
     - "Restart Buildkite Agent and Register for Login"
+
+- name: Reverting homebrew state back to HEAD
+  command: git checkout master
+  args:
+    chdir: /usr/local/Homebrew
+
+- name: Refresh homebrew to original state
+  command: /usr/local/bin/brew update
```

Comment:
If we do this, don't do the `git checkout ...` immediately preceding it?

Reply:
`brew update` does not check out the right branch so both steps are needed. I'm running brew update to validate that brew is working again.